### PR TITLE
Update release notes for 11.2

### PR DESCRIPTION
**Pull request by anitaa1990**
Update release notes for 11.2

### CODE REVIEW

**Comment by ealeksandrov** [comment]
@anitaa1990 @oguzkocer we're releasing a new public-facing feature (product previews) in 11.2: https://github.com/woocommerce/woocommerce-ios/pull/8102.
We may want to add it in the release notes.
Sorry for the late notice!

**Comment by anitaa1990** [comment]
Hey @ealeksandrov, from what I understand the Preview option is run as part of an experiment so it won't be visible to all users. That's how it was released on android. Is that the case for iOS? If so, I won't include it in the release notes.

**Comment by ealeksandrov** [comment]
No, it's being released to all users without A/B experiment.
And we don't have that feature implemented on Android yet.

**Review by oguzkocer — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 11.2
+For this release, we focused on polishing a few things and taking care of minor bugs that our team found, including the option to preview your products before publishing them. Don’t be fooled – we are working on some pretty cool stuff for you all! Keep sharing feedback 💜
```
Comment (oguzkocer) [inline CHANGELOG.md:5]: @anitaa1990 I just wanted to let you know that App Store Connect didn't allow the 💜 character and I had to replace it with `!` in https://github.com/woocommerce/woocommerce-ios/pull/8153 before submitting `11.2`.

**Pull request merged**